ci-bootstrap/README.md

```diff
@@ -0,0 +1,15 @@
+# Resource for setting up role in a new AWS account.
+
+When setting up a new infrastructure for the lambda, first run deploy.sh script to create a correct role with correctly assigned policies.
+
+Replace ENVIRONMENT variable ``` <accountname> ``` with sandbox, cicd, staging or prod
```

Comment:
Inline code chunks in markdown should use single backticks. The triple backticks are used for multiline preformatted code blocks

Reply:
Didn't know that, thanks